Milione/graph.gv

```diff
@@ -0,0 +1,41 @@
+# Place the cursor inside "graph" to get some refactoring options
+
+digraph {
+    # Greek letters/hypothetical links
+    omega[color="grey", label="Ω"]
+    alpha[color="grey", label="α"]
+    beta[color="grey", label="β"]
+    delta[color="grey", label="δ"]
+    delta¹[color="grey", label="δ¹"]
+    delta²[color="grey", label="δ²"]
+    delta³[color="grey", label="δ³"]
+    gamma[color="grey", label="γ"]
+    epsilon[color="grey", label="ε"]
+
+    omega -> alpha;
+    omega -> beta;
+
+    alpha -> gamma;
+    gamma -> V;
+    alpha -> delta;
+    delta -> Fr;
+    Fr -> delta²;
+    delta² -> l;
+    delta² -> F;
+    l -> L;
+
+    delta -> delta¹;
+    delta¹ -> delta³;
+    delta³ -> TA;
```

Comment:
I'm wondering: TA, VA and VB are single manuscripts ? What are they ?

Reply:
La redazione VB del Milione di Marco Polo è composta da tre codici che attestano un volgarizzamento veneziano della metà del Quattrocento: di essi, il manoscritto Donà dalle Rose 224 del Museo Civico Correr di Venezia funge da base dell’edizione che propongo.

Reply:
OK, so I guess TA, VA, VB and Fr are actually versions, or, to say it better, archetypes of the said versions. So they are «lost» but represent several extant mss that are not printed here.